packages/netlify-cms-core/src/actions/config.js

```diff
@@ -118,10 +118,12 @@ export function applyDefaults(config) {
             );
           }
 
-          if (!collection.has('sortableFields')) {
+          if (!collection.has('sortableFields') && !collection.has('sortable_fields')) {
             const backend = resolveBackend(config);
             const defaultSortable = selectDefaultSortableFields(collection, backend);
             collection = collection.set('sortableFields', fromJS(defaultSortable));
+          } else if (collection.has('sortable_fields')) {
+            collection.set('sortableFields', collection.get('sortable_fields'));
```

Comment:
We would like to use `sortable_fields` internally too to avoid confusing

Reply:
Alright, sure. Will update this as well along with the other camelCase config